libs/@hashintel/ds-components/README.md

```diff
@@ -0,0 +1,31 @@
+# @hashintel/ds-components
+
+Components for HASH refractive design system, built using React, TypeScript, ArkUI, and PandaCSS.
+
+## Synchronization with Figma
+
+These components rely on variables and styles defined in Figma, which are synchronized using the Figma and ArkUI MCP servers.
+
+Design Tokens are sourced from the `@hashintel/ds-theme` package, which is generated from Figma variables to a Panda CSS preset.
```

Comment:
```suggestion
Design Tokens are sourced from the `@hashintel/ds-theme` package, which is generated from Figma variables mapped to a Panda CSS preset.
```
I think this is what you're saying here, but not 100% sure.

Reply:
I think to encapsulate concerns inline it might also be worth us just saying, here, that "Design Tokens are sourced from the `@hashintel/ds-theme` package" and leaving it at that. The specifics of how that works could and should be documented inside that separate package's README.